hotsos/core/plugins/storage/ceph.py

```diff
@@ -31,6 +32,16 @@
 
 CEPH_LOGS = "var/log/ceph/"
 
+CEPH_EOL_INFO = {
+    'quincy': datetime(2032, 4, 30),
```

Comment:
Tthese are Ubuntu EOL dates, right?

ceph upstream has a much shorter cycle: https://docs.ceph.com/en/quincy/releases/index.html

I don't know if we (can) actually support that far out once upstream stopped accepting patches. So I am wondering if we need to use upstream dates. @hillpd comments?

Reply:
That was actually something that came to mind while implementing this. Waiting for input from @hillpd or maybe @dosaboy to get some answers on this.

Reply:
Had a discussion in Ceph OH.

hillpd thinks we should use what our service description dates. But there's some complication there
as to what the right EOL date is for a Ceph pkg:

    If it's UCA/Ceph package then use UCA EOL date.
    If it's Ubuntu/Ceph package then use Ubuntu EOL date (which is what's currently set here in this PR).

Reply:
@tpsilva @pponnuvel sorry for delayed response. I agree that using service description dates makes sense and since the majority of users will be using UCA for non-lts versions it makes sense to track UCA dates.